### ISSUE TITLE
BUG: Some ships don't load for client in MP

### ISSUE THREAD

**Issue by Vicomt** [tsunamayo/Starship-EVO#3157]
### Short Description of the issue:
when connecting a client to a host world, some of the ships in the local area do not appear. The chunk count also appears to be considerable lower on the client than on the host.

### Steps to reproduce the issue:
join a MP world with a few ships in the local area (world attached below)

### Environment
- Starship EVO version: 20w45a
- Graphics Card: GTX1070 (host)

### Additional information:

[vic mp test 2.zip](https://github.com/tsunamayo/Starship-EVO/files/5483605/vic.mp.test.2.zip) (world)
[output_log.zip](https://github.com/tsunamayo/Starship-EVO/files/5483609/output_log.zip) (host output log)
client output log to be attached by Oojimaflip

**Comment by oojimaflip** [comment]
![20201103191935_1](https://user-images.githubusercontent.com/33450683/98031494-7bc5d100-1e0a-11eb-8880-b51c6d8eec8d.jpg)
view from the same position as client sees it.
chunk loading seems to stall at 10%

output log for session:
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5483643/output_log.txt)

**Comment by Vicomt** [comment]
interestingly, my chunk count is roughly 10% as well - but I assume that's because I've got more RAM?

**Comment by Vicomt** [comment]
I did some ship re-arranging, then we reloaded. a different set of ships was visible for the client this time.

(host view)
![20201103195630_1](https://user-images.githubusercontent.com/7197427/98034321-b6ca0380-1e0e-11eb-9497-c465569c7315.jpg)

**Comment by oojimaflip** [comment]
![20201103195626_1](https://user-images.githubusercontent.com/33450683/98034415-deb96700-1e0e-11eb-911d-3fde26474455.jpg)
from the same position

**Comment by StarrKiss** [comment]
Can confirm this happens too with me

**Comment by tsunamayo** [comment]
@Vicomt okay did you guys waited a bit? It will take a bit of time for large ships. I see a few error in the log, I will fix them and we will see if that fixes the issue. thanks for helping me on mp guys!

**Comment by Vicomt** [comment]
yes, we waited. we played a good couple of hours in one session and they never showed up.

**Comment by tsunamayo** [comment]
Hi guys, so there is a new build 20w45b. I did a few fixes regarding the issues in your log, I am not sure if it fixes the spawn issues though, if you can tell how it went. Thanks

**Comment by tsunamayo** [comment]
@Vicomt @StarrKiss @oojimaflip I dont know if you manage to try the latest build, to confirm if the bug is there or not! Thanks all
